library/entity/villager/src/testmod/resources/quilt.mod.json

```diff
@@ -0,0 +1,23 @@
+{
+  "schema_version": 1,
+  "quilt_loader": {
+    "group": "org.quiltmc.qsl.entity",
+    "id": "quilt_villager_testmod",
+    "version": "1.0.0",
+    "metadata": {
```

Comment:
Since I can't really comment on the file itself: you don't need an icon.png for testmods